app/src/main/java/com/example/diceroller/MainActivity.kt

```diff
@@ -22,11 +32,21 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 @Preview
-fun DiceRollerApp() {
+fun DiceRollerApp(){
     DiceWithButtonAndImage()
 }
 
 @Composable
-fun DiceWithButtonAndImage() {
-
-}
+fun DiceWithButtonAndImage(modifier: Modifier = Modifier
+    .fillMaxSize()
+    .wrapContentSize(Alignment.Center))
```

Comment:
```suggestion
fun DiceWithButtonAndImage(
modifier: Modifier = Modifier
    .fillMaxSize()
    .wrapContentSize(Alignment.Center))
```
When we have more than 1 line of parameters, we usually split the parentheses from the parameters